static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -478,12 +478,11 @@ $block-padding-right: $gs-gutter;
 .dropdown--live-feed {
     border-top: 0;
 
+    $content-gutter: $gs-gutter/2;
+
     @include mq($until: desktop) {
-        margin-right: $gs-gutter*-1;
-        margin-left: $gs-gutter*-1;
-        padding-right: $gs-gutter;
-        padding-left: $gs-gutter;
-        padding-top: $gs-baseline*4;
+        margin-right: ($content-gutter)*-1;
```

Comment:
slightly unsure about abstracting `$gs-gutter` away? wonder if it makes it harder to skim?

i think it's also useful to add white space around operators in sass, so you can see immediately that it's not a name, it's an operation